py/torch_tensorrt/dynamo/_compiler.py

```diff
@@ -91,6 +148,7 @@ def compile(
     custom_engine_cache: Optional[BaseEngineCache] = _defaults.CUSTOM_ENGINE_CACHE,
     use_explicit_typing: bool = _defaults.USE_EXPLICIT_TYPING,
     use_fp32_acc: bool = _defaults.USE_FP32_ACC,
+    enable_cross_compile_for_windows: bool = _defaults.ENABLE_CROSS_COMPILE_FOR_WINDOWS,
```

Comment:
Does this work? if I use `dynamo.compile` + `enable_cross_compile_for_windows`, will it succeed or hit the issues with setup? We probably want to figure out another way do this this through since this is a user API

Reply:
Same question - To avoid setting up, this option should disable setting up engines by checking the TorchTensorRTModules don't setup engines during initialization. is this happening ?

Reply:
removed this settings, also added check make sure
dynamo.compile + enable_cross_compile_for_windows will raise the error:
please use the dynamo.cross_compile_for_windows() api.